### PR TITLE
update auto architecture document in GitHub for 2023.0 release

### PR DESCRIPTION
**Pull request by wangleis**
### Details:
 - *Update auto architect document in GitHub for 2023.0 release*
 - *...*

### Tickets:
 - *100527*

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: src/plugins/auto/docs/architecture.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+## Specific Features of AUTO plugin
+
+### Accelerating First Inference Latency (FIL)
+
+Compiling the model to accelerator-optimized kernels may take some time. When AUTO select one accelerator, AUTO can start inference with the CPU of the system by default, as it provides very low latency and can start inference with no additional delays. While the CPU is performing inference, AUTO continues to load the model to the device best suited for the purpose and transfers the task to it when ready. 
+
+Below is the example of CPU acceleration during GPU compilation. 
+![alt text](https://docs.openvino.ai/latest/_images/autoplugin_accelerate.svg "AUTO cuts first inference latency (FIL) by running inference on the CPU until the GPU is ready")
+
+User can disable this acceleration feature by excluding CPU from the priority list or disable ov::intel_auto::enable_startup_fallback.The default value of ov::intel_auto::enable_startup_fallback is true.
```
Comment (kblaszczak-intel) [inline src/plugins/auto/docs/architecture.md:27]:
```suggestion
The user can disable this acceleration feature by excluding CPU from the priority list or disabling `ov::intel_auto::enable_startup_fallback`. Its default value is `true`.
```

File: src/plugins/auto/docs/architecture.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+
+### Inference on Multiple Devices
+
+The ov::hint::performance_mode property enables you to specify a performance option for AUTO to be more efficient for particular use cases. And the CUMULATIVE_THROUGHPUT hint enables running inference on multiple devices for higher throughput.
```
Comment (kblaszczak-intel) [inline src/plugins/auto/docs/architecture.md:31]:
```suggestion
The `ov::hint::performance_mode` property enables you to specify a performance option for AUTO to be more efficient for particular use cases, while the `CUMULATIVE_THROUGHPUT` hint enables running inference on multiple devices at once for higher throughput.
```

File: src/plugins/auto/docs/architecture.md [hunk 3]
```diff
@@ -1,6 +1,42 @@
...
+Below is the example of CPU acceleration during GPU compilation. 
+![alt text](https://docs.openvino.ai/latest/_images/autoplugin_accelerate.svg "AUTO cuts first inference latency (FIL) by running inference on the CPU until the GPU is ready")
+
+User can disable this acceleration feature by excluding CPU from the priority list or disable ov::intel_auto::enable_startup_fallback.The default value of ov::intel_auto::enable_startup_fallback is true.
+
+### Inference on Multiple Devices
+
+The ov::hint::performance_mode property enables you to specify a performance option for AUTO to be more efficient for particular use cases. And the CUMULATIVE_THROUGHPUT hint enables running inference on multiple devices for higher throughput. 
+
+With CUMULATIVE_THROUGHPUT, AUTO loads the network model to all available devices in the candidate list, and then runs inference on them based on the devices priority.
```
Comment (kblaszczak-intel) [inline src/plugins/auto/docs/architecture.md:33]:
```suggestion
With CUMULATIVE_THROUGHPUT, AUTO loads the model to all available devices in the candidate list. Then, it runs inference, choosing devices based on their priority.
```

This is more than just rewording, see if it makes sense to make this change or not.

**Review by kblaszczak-intel — COMMENTED** [review]
I've added some suggestions. In one case I'm not sure if the meaning is still perfectly fine.

**Review by kblaszczak-intel — APPROVED** [review]

**Pull request merged**